### DISCUSSION TITLE
Is there a way to bind to "node events"?

### DISCUSSION THREAD

**Discussion by karmaral** [ueberdosis/tiptap#2982] [General Questions]
Hello!
I'm trying to hook a node's content to a state variable. So far the only way I found to achieve this was to create a Node View using a React component as the content and handling some of the stuff there. But it doesn't seem too elegant.
Is there something like
`onNodeUpdated` or `onNodeCreated` in the node constructor?

Or should I listen for all editor transactions and filter them to see if they match my node? That feels a bit low level.
I'd gladly appreciate some pointers. Thanks!

**Comment by emroot** [comment]
Really curious about this too. Did you find a solution?
